docker-setup.sh

```diff
@@ -6,12 +6,13 @@ set -e
 
 pull_docker_snapshot() {
   project="${1?project name required}"
+  stack_version="$2"
```

Comment:
since this function already works with the ELASTIC_STACK_VERSION variable, calling this other one "stack_version" doesn't convey the purpose or why it's needed, compared to ELASTIC_STACK_VERSION.

Maybe:

```suggestion
  stack_version_arg="$2"
```

or `orig_stack_version_arg`

Alternatively we can come up with another name altogether for the "8.next" / "8.current" / etc. references, like ELASTIC_STACK_VERSION_ALIAS

Reply:
I like the `_ALIAS` idea, that suggests to me more clearly that transformations may take place in the script. I'll update that. Thanks!